packages/pigeon/test/cpp_generator_test.dart

```diff
@@ -875,6 +882,12 @@ void main() {
           code,
           contains(
               'const auto* an_object_arg = &(std::any_cast<const ParameterObject&>(std::get<flutter::CustomEncodableValue>(encodable_an_object_arg)));'));
+      // "Object" requires no extraction at all since it has to use
+      // EncodableValue directly.
+      expect(
+          code,
+          contains(
+              'const auto* a_generic_object_arg = &encodable_a_generic_object_arg;'));
```

Comment:
These new unit tests aren't strictly necessary, since it was already failing compilation, but since we have this suite of extraction format tests already I thought it would be useful to include them to have fast-test coverage of this special case.